mock.exception.message = This is a mock exception {0} !
unauthorized.message = Login failed!